contrib/epee/include/file_io_utils.h

```diff
@@ -115,6 +141,14 @@ namespace file_io_utils
 	}
 
 	inline
+		std::string file_to_string(const std::string& path)
+	{
+		std::string target_string;
+		epee::file_io_utils::load_file_to_string(path, target_string);
+		return target_string;
+	}
```

Comment:
That seems pointless since it's just one call, and also ignores the return code, so makes things easier to get wrong. While the particular usage in your patch does not seem to depend on that, putting this in a generic file ensures it'll be used that way at some point.